### ISSUE TITLE
Snapshot URL: doesn't upload to google drive

### ISSUE THREAD

**Issue by jasperza** [ccrisan/motioneyeos#2252]
What is the URL or API of the "Take a snapshot" button shown below.
NOTE: the "snapshot" url  "..../picture/1/current" does not have the same effect, it only displays a snapshot in the browser. 

1. I want to use an API to trigger motioneye to take a photo and save it. There is a small icon on the bottom right of the screen which allows the user to trigger taking a snapshot, however I want to do this programatically, what is the URL to do this?
2. I also want to provide the filename , or part of the filename, e.g. a unique number that my program generates, which i can reference to other sensor data later when I download the images , it it possible to provide this?  alternatively, does the snapshot URL return the filename that was created?
![button snapshot](https://user-images.githubusercontent.com/12615954/72065709-8812f780-32e7-11ea-9379-20304a1da249.jpg)

**Comment by Bobo-amg** [comment]
You can use curl to capture a snapshot with {IP}:{port0}/{camid}/current (eg curl  -o newimg.jpg 10.1.1.253/picture/1/current/) will take a snapshot and save it your local drive.

**Comment by jasperza** [comment]
Yes but does that also capture it to the rpi? I am calling it from an IoT device that doesn't have a browser, Local drive or human.

**Comment by Bobo-amg** [comment]
Yes it does. It will save the image to the device that is running the curl command (which could be the pi camera). So I have a pi server that runs the curl command remotely on the 253 camera (also a pi). The image is saved on the server.

I have made this even easier by using Node Red and sending images to Telegram.

**Comment by jasperza** [comment]
Thanks for your reply, I did call the snapshot URL "http://10.0.0.244/picture/1/current/" and it doesn't create the file on the motioneye device and it doesn't upload it to google drive

The behavior is different when I click the "snapshot" button: it _does_ save it on the motioneye device upload it to google drive. 

These settings work when clicking the "snapshot" button but dont work when calling the snapshot URL
![image](https://user-images.githubusercontent.com/12615954/72786768-1fa50e00-3c37-11ea-9d8b-3aaeae7bf420.png)
